src/Proto.Cluster/Cluster.cs

```diff
@@ -103,8 +103,9 @@ private async Task BeginStartAsync(bool client)
 
         public async Task ShutdownAsync(bool graceful = true)
         {
+            await System.ShutdownAsync();
```

Comment:
Not sure what is the optimal order here.
Maybe it should be the other way around. but this solves the problem for now